lib/pack.js

```diff
@@ -67,9 +74,10 @@ class Pack extends BaseCommand {
     for (const { arg, filename, manifest } of manifests) {
       const tarballData = await libpack(arg, this.npm.flatOptions)
       const pkgContents = await getContents(manifest, tarballData)
+      const tarballFilename = path.join(this.npm.config.get('pack-destination'), filename)
```

Comment:
isn't it better to stick with `path.resolve` instead of `path.join` here?

Reply:
Sure